tests/interop/create_ci_badge.py

```diff
@@ -0,0 +1,84 @@
+import json
```

Comment:
Should we (event at a later time) move this one to the common parts of the qe tests? Asking as it seems pretty generic

Reply:
makes sense